src/api/BorrowerStandardPortalGeneralSettingsApi.ts

```diff
@@ -27,7 +27,7 @@ export interface BorrowerStandardPortalGeneralSettingsApi {
 }
 
 export class BorrowerStandardPortalGeneralSettingsRestApi implements BorrowerStandardPortalGeneralSettingsApi {
-  protected path = '/borrowers/standard-portals/general-settings';
+  protected path = 'borrowers/standard-portals/general-settings';
```

Comment:
not sure why it's putting / to each api call is better than have it in one const path?

Reply:
fixed